dev-tools/cmd/dashboards/export_dashboards.go

```diff
@@ -157,42 +46,55 @@ func main() {
 	flag.Parse()
 	log.SetFlags(0)
 
-	transCfg := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true}, // ignore expired SSL certificates
+	elements := strings.Split(*kibanaURL, "://")
+	if len(elements) != 2 {
+		log.Fatalf("Invalid Kibana URL format")
```

Comment:
Maybe we could rely on the standard library to do the url parsing validation.
But we would have to check if the scheme is an empty string.


https://play.golang.org/p/UpHEGT6yRdn